src/status_im2/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -0,0 +1,63 @@
+(ns status-im2.contexts.wallet.send.input-amount.view
+  (:require
+    [clojure.string :as string]
+    [quo.core :as quo]
+    [quo.theme :as quo.theme]
+    [react-native.core :as rn]
+    [reagent.core :as reagent]
+    [status-im2.contexts.wallet.common.temp :as temp]
+    [status-im2.contexts.wallet.send.input-amount.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn- view-internal
+  [_]
+  (let [token                 :eth
+        currency              :usd
```

Comment:
I think currency can be taken from `{:keys [currency]} (rf/sub [:profile/profile])

Reply:
Fixed